### PR TITLE
Fixing windows RANS compilation

### PR DESCRIPTION
**Pull request by roigcarlo**
**Description**
Fixes several compilation problems in win.

also @sunethwarna , please consider grouping the test_utilities in a class and use the functions as static members rather than having them inside a namepsace.

**Changelog**
- Added missing exports to some templated utilities
- Changed size_t to int to prevent problems in windows.

### CODE REVIEW

**Comment by philbucher** [comment]
maybe add the RANSApp also to the Win-CI?

**Comment by philbucher** [comment]
now there is a conflict

**Comment by sunethwarna** [comment]
thanks @roigcarlo :)... I'll do as you suggest :)

**Comment by roigcarlo** [comment]
@philbucher I fail to understand whats happening with win, did I made any wrong change in the CI?

**Comment by philbucher** [comment]
I have no idea either 
It fails when trying to install, so after compiling

Can you try locally?
I looked at the code but it looks correct to me

**Comment by roigcarlo** [comment]
I suspect is a problem with the CI cache, I disabled the app, if it works I will re-enable it in another PR with a clean cache.

**Comment by roigcarlo** [comment]
I suspected right

File: .github/workflows/configure.cmd [hunk 1]
```diff
@@ -25,6 +25,7 @@ set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%KRATOS_APP_DIR%\StatisticsApplicat
 set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%KRATOS_APP_DIR%\SwimmingDEMApplication;
 set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%KRATOS_APP_DIR%\ShapeOptimizationApplication;
 set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%KRATOS_APP_DIR%\ConstitutiveLawsApplication;
+REM set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%KRATOS_APP_DIR%\RANSApplication;
```
Comment (sunethwarna) [inline .github/workflows/configure.cmd:28]: can we revert this back on? Afterwards I can approve. Thanks a lot @roigcarlo. I'll change the test_utilities.h in a sepearte PR.

**Review by sunethwarna — CHANGES_REQUESTED** [review]

File: applications/RANSApplication/custom_processes/rans_line_output_process.h [hunk 2]
```diff
@@ -35,7 +35,7 @@ namespace LineOutputProcessUtilities
 using NodeType = ModelPart::NodeType;
 using GeometryType = ModelPart::ElementType::GeometryType;
 using SizeType = std::size_t;
-using IndicesVector = std::vector<SizeType>;
+using IndicesVector = std::vector<int>;
```
Comment (sunethwarna) [inline applications/RANSApplication/custom_processes/rans_line_output_process.h:38]: Just curious, why using `std::size_t` cause a problem in windows compilation?

File: .github/workflows/configure.cmd (hunk 1 above)
Comment (roigcarlo) [reply .github/workflows/configure.cmd:28]: Saldy no, we have to merge with this disabled, once its merge I will activate it in another PR. Its causing problems with the cache dor some reason...

File: applications/RANSApplication/custom_processes/rans_line_output_process.h (hunk 2 above)
Comment (roigcarlo) [reply applications/RANSApplication/custom_processes/rans_line_output_process.h:38]: It gets evaluated to something wierd like `uint_64`  and then the types for the mpi interface are not deduced correctly

**Review by sunethwarna — APPROVED** [review]

**Pull request merged**